### DISCUSSION TITLE
I can't get past the Photoprism login screen

### DISCUSSION THREAD

**Discussion by dinofours** [photoprism/photoprism#4601] [Q&A]
My photoprism suddenly stopped login me in. I enter my correct user name and password and it doesn't give a wrong credentials error. It loads the screen, but back to the login page. Almost like it just refreshes the screen.

I don't know what to do. I've already installed and uninstalled the plug in on openmediavault, but it still the same.

r/OpenMediaVault - I can't get past the Photoprism login screen

Is there any way to completely reset photoprism? Including all config files, passwords, etc?

**Comment by lastzero** [comment]
If you are unable to log in, we recommend that you go through the following troubleshooting checklist:

- https://docs.photoprism.app/getting-started/troubleshooting/#cannot-log-in

Additional notes, e.g. how to change the password from a terminal, can be found below.

**Reply by dinofours** [reply]
Hi, thanks for that! But I think you might have forgotten to add the additional notes on how to change the password from a terminal?

**Reply by omerdduran** [reply]
Hi! I think you missed it. It's number five on the checklist.  

https://docs.photoprism.app/user-guide/users/cli/#changing-a-password

**Reply by dinofours** [reply]
Hi @omerdduran and @lastzero.

I went through the documentation, but I'm afraid it doesn't apply to my set up since I'm using photoprism as the plugin on openmediavault, not through a docker and portainer.
I can access my box through ssh though. Maybe there's something I can do through there?

**Reply by omerdduran** [reply]
In OpenMediaVault, PhotoPrism runs inside a Podman container.

Terminal access should therefore work by connecting via SSH to the machine running OpenMediaVault and then running the following commands.

(1) To verify that PhotoPrism is running:

```bash
sudo podman ps
```

You should see photoprism-app (or similar) among the results.

(2) To open a PhotoPrism terminal session (assuming "photoprism-app" is the container name):

```bash
sudo podman exec -it photoprism-app /bin/bash
```

From this terminal you should be able to run the PhotoPrism CLI commands as described in the documentation at https://docs.photoprism.app/user-guide/users/cli/.

I hope this helps! Please note, however, that this is a third-party integration, so it may be better to ask the OMV community if this general advice doesn't solve your problem.

**Reply by lastzero** [reply]
We have added this to our Getting Started Guide in case someone has a similar question:
- https://docs.photoprism.app/getting-started/nas/openmediavault/#opening-a-terminal

**Comment by dinofours** [comment]
Hi Ömer,

Thanks for reaching out and trying to help. I fixed the issue by deleting
all the folders and files associated with it photoprism and then
reinstalled the plugin and reconfigured everything.

It's back working again. But if it somehow happens ahappy, I'll follow your
steps and let you know.

Thank you!

**Comment by dinofours** [comment]
I have another question actually.

Is there any way to create a second instance of photoprism on another port?

**Reply by lastzero** [reply]
If you use Podman or Podman Compose, it's possible if you change the configuration accordingly. Since this is a third-party integration, it's best to ask the OMV community if/how this is possible from the web interface.

**Comment by dinofours** [comment]
Hi Michael,

Sorry to bother you agai, but I have another question.

Using photoprism through the openmediavault add on, I don't have access to
change all environmental variable. Is there any way that I can enabled
automatic imports using the add on?

Thanks!

**Reply by omerdduran** [reply]
Hello. Even if it seems that you do not have access to the UI side, you can actually edit it. For this, you can take a look at the following content. 

https://docs.openmediavault.org/en/stable/various/advset.html#environmental-variables
https://forum.openmediavault.org/index.php?thread/49417-photoprism-plugin-more-options-please/

**Comment by dinofours** [comment]
That's perfect. Thanks!
